frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java

```diff
@@ -56,13 +60,18 @@ public void actionPerformed(AnActionEvent anActionEvent) {
             /* message */ getString("action.GitMachete.OpenMacheteFileAction.notification.title.cannot-open"))))
         .getOrNull();
 
-    getGraphTable(anActionEvent).queueRepositoryUpdateAndModelRefresh();
-
     if (macheteFile == null) {
-      VcsNotifier.getInstance(project).notifyError(/* displayId */ null,
+      val errorWithDiscover = VcsNotifier.STANDARD_NOTIFICATION.createNotification(
           /* title */ getString("action.GitMachete.OpenMacheteFileAction.notification.title.machete-file-not-found"),
           /* message */ getString("action.GitMachete.OpenMacheteFileAction.notification.message.machete-file-not-found")
-              .fmt(gitDir.getPath()));
+              .fmt(gitDir.getPath()),
+          NotificationType.ERROR);
+      errorWithDiscover.addAction(NotificationAction
```

Comment:
Note there is no `expire()`. It's hard to track the notification status and all the cases that could make it expired. On the other hand the action is simply a call to discover action - one see the suggested dialog first, must accept it. So even though the action will remain available as long as the notification lives there is no big deal imho. wdyt @PawelLipski ?

Reply:
Makes sense... maybe pls put the above musings into a code comment!